directus/Caddyfile

```diff
@@ -0,0 +1,28 @@
+# change to the hostname you want to bind to
+localhost:3000
+
+# change to correct document root
+# in this example config directus is assumed to be root
+# you may need to adjust rewrites according to your own setup
+root /var/www/directus
+
+# Set the path to the php-fpm process.
```

Comment:
I actually mean if you run directus in a subdirectory which is quite common since it's a headless CMS